ngrinder-frontend/src/js/components/perftest/detail/Config.vue

```diff
@@ -231,38 +232,21 @@
 
     @Component({
         name: 'config',
-        props: {
-            testProps: {
-                type: Object,
-                required: true,
-            },
-            config: {
-                type: Object,
-                required: true,
-            },
-        },
         components: { DurationSlider, TargetHostInfoModal, ControlGroup, InputAppend, InputPrepend, InputPopover, HostModal, Select2, RampUp },
     })
     export default class Config extends Mixins(Base, MessagesMixin) {
         @Inject() $validator;
 
-        test = {
-            param: '',
-            region: '',
-            testName: '',
-            agentCount: 0,
-            rampUpInitCount: 0,
-            rampUpStep: 0,
-            rampUpInitSleepTime: 0,
-            rampUpIncrementInterval: 0,
-            ignoreSampleCount: 0,
-            runCount: 0,
-            processes: 0,
-            threads: 0,
-            vuserPerAgent: 0,
-            samplingInterval: 2,
-            threshold: 'D',
-        };
+        @Prop({ type: Object, required: true })
+        testProp;
+
+        @Prop({ type: Array, required: true })
+        scriptsProp;
```

Comment:
Why is Prop post fix necessary?

Reply:
This child component copies `props` into its own `data` to ensure that not modify parent's property.
`testProp -> test, scriptsProp -> scripts`